rosa_getting_started/rosa-aws-prereqs.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 {product-title} (ROSA) provides a model that allows Red Hat to deploy clusters into a customer’s existing Amazon Web Service (AWS) account.
 
-You must ensure that the prerequisites are met before installing ROSA.
+You must ensure that the prerequisites are met before installing ROSA. This requirements document does not apply to AWS Security Token Service (STS). If you are using STS, see the link:https://docs.openshift.com/rosa/rosa_getting_started_sts/rosa-sts-aws-prereqs.html[STS-specific requirements].
```

Comment:
Could this be an xref? 

(I know nothing about how the ROSA docs are put together, so this might be a dumb question.)